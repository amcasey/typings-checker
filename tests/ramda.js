var R = {
    isArrayLike: function (x) { return true; },
};
(function () {
    // $ExpectType boolean
    R.isArrayLike('a');
    // $ExpectType boolean
    R.isArrayLike([1, 2, 3]);
    // $ExpectType boolean
    R.isArrayLike([]);
});
