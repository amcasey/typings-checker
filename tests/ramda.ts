const R = {
  isArrayLike(x: any) { return true; },
};

() => {
  // $ExpectType boolean
  R.isArrayLike('a');
  // $ExpectType boolean
  R.isArrayLike([1,2,3]);
  // $ExpectType boolean
  R.isArrayLike([]);
};
