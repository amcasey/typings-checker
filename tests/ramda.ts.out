tests/ramda.ts: 3 / 3 checks passed.
